pkgs/tools/audio/beets/common.nix

```
@@ -23,6 +23,10 @@
 , version
 , pluginOverrides ? { }
 , disableAllPlugins ? false
+
+  # tests
+, runCommand
+, pkgs
```

This increases evaluation time without need. I think this should work:

```suggestion
, beet
```